src/main/java/de/False/BuildersWand/events/WandEvents.java

```diff
@@ -265,7 +272,7 @@ private int getItemCount(Player player, Block block, ItemStack mainHand)
             }
             Material itemMaterial = itemStack.getType();
 
-            if (!itemMaterial.equals(blockMaterial) || block.getData() != itemStack.getDurability())
+            if (!itemMaterial.equals(blockMaterial) || block.getData() != itemStack.getData().getData())
```

Comment:
This was my first attempt to fix the #30 issue. Later I realized, this wasn't the problem.
I still let it there, because "Never change a running system"!